ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -561,6 +561,10 @@ QtObject {
         return transaction.txType
     }
 
+    function addressesEqual(address1, address2) {
```

Comment:
This is referenced in https://github.com/status-im/status-desktop/pull/16406/files#diff-49a543b84eef8685a0154567966dc0bfb8238b733a8beeb4149988a75604f4b8R545 but was missing (forgotten in the component itself)